userdocs/src/usage/nodegroup-managed.md

```diff
@@ -116,7 +116,7 @@ managedNodeGroups:
 
 ```yaml
 # cluster.yaml
-# A cluster with an unmanaged nodegroup and two managed nodegroups.
+# A cluster with a managed nodegroup with customization.
```

Comment:
We should mention that the example uses a custom AMI. 
```suggestion
# A cluster with a managed nodegroup using a custom AMI. 
```

Reply:
I used "customization" in general since it uses other custom features besides the custom AMI, like security groups, volumeName, volumeEncrypted, maxPodsPerNode, overrideBootstrapCommand, and disableIMDSv1.

Let me know if you still prefer custom AMI if that's the main aspect to highlight!